examples/shapes_to_labels.py

```diff
@@ -60,3 +60,7 @@
     labels = layer.data.to_labels([512, 512]).transpose(1, 0)
     labels_layer = viewer.add_labels(labels, name='labels')
     labels_layer.visible = False
+
+    svg = layer.to_svg()
+    with open('shape.svg', 'w') as file:
```

Comment:
sorry, I missed this earlier, but you should preferably use `f` instead of `file` as it is a built-in name

Reply:
i am going to comment out the file writing and just generate the string